hittingEndPoint.vb

```diff
@@ -59,10 +63,11 @@ Module hittingEndPoint
 
     Public Function callSheetsAPI(ByRef fileId As String, ByRef userClientId As String , ByRef userClientSecretId As String) As List(Of String)
         Dim lines As List(Of String) = Nothing
+        Const URI As String = "https://docs.google.com/spreadsheets/vbaprocessfile?fid=" + fileId

```

Comment:
Be consistent across code base. Const string here is in all capital 'URI' but on line 11-14 it is camel case with a prefix underscore.

Reply:
Done.